forwards_paper.tex

```diff
@@ -148,71 +133,99 @@ \section*{Introduction}
 so we discard all information irrelevant to the genetic history
 of the \emph{sampled} individuals,
 which results in considerable savings.
-
 Another way of representing this same information
 is known as the \emph{ancestral recombination graph}, or {ARG} \citep{griffiths1997ancestral},
 which has been the subject of substantial study
 under the assumptions of coalescent theory \citep{wiuf1997number,wiuf1999ancestry,marjoram2006coalescent,wilton2015smc}.
 
+\plr{Quick review of simulation software, esp that other one that does the same thing.}
+
+\plr{Make clear that the main product here is the \msprime{} interface,}
+part of which is the tables schema.
+
 
 %%%%%%%%%%%%%%%%%%%%%%
-\section*{Methods}
+\section*{Results}
+
+\plr{Order is intro-results-discussion-methods.}
+
+Below, we first describe the conceptual and algorithmic foundations for the method:
+(a) the new \emph{tables} format for storing tree sequences;
+(b) an algorithm to output tree sequence information on the fly
+    from a forwards-time simulation;
+and (c) a method to \emph{simplify} a tree sequence, i.e., remove redundant information.
+This is followed by benchmarking comparisons
+with real, forwards-time simulations.
 
-\plr{Reminder of what we need to know in the end (the trees), and
-    quick review of \msprime{} methods: sparse trees, tree differences.}
 
 %%%%%%
-\subsection*{Tree sequences}
-
-A tree sequence is an efficient encoding for a sequence of correlated trees such as
-is produced by recombination. The encoding is efficient because branches that are
-shared by adjacent trees are stored once, rather than repeatedly for each tree.
-The topology of a tree sequence is defined via a set of \emph{nodes} and
-\emph{edges}. A node in a tree sequence refers to a distinct ancestor and
-corresponds to the vertices in the individual genealogies along the sequence.
+\subsection*{Recording a tree sequence}
```

Comment:
I think this section is not just about tree sequences but how to record them in the four tables.